docs/hog_scenarios.md

```diff
@@ -18,6 +18,8 @@ these workloads will use a predetermined amount of resources for a specified dur
 |`namespace`| string                                                                                                                                                                                                                                                                                                                                                  | the namespace where the stress workload will be deployed                                                                                                                                                                                                                                                                              |
 |`node-selector`| string (Optional) | defines the node selector for choosing target nodes. If not specified, one schedulable node in the cluster will be chosen at random. If multiple nodes match the selector, all of them will be subjected to stress. If number-of-nodes is specified, that many nodes will be randomly selected from those identified by the selector. |
 |`number-of-nodes`| number (Optional) | restricts the number of selected nodes by the selector|
+|`taints`| list (Optional) default [] | list of taints for which tolerations need to created. Example: ["node-role.kubernetes.io/master:NoSchedule"]|
```

Comment:
This change needs to be added to https://github.com/krkn-chaos/website/tree/main/content/en/docs/scenarios/hog-scenarios/cpu-hog-scenario.